src/version.h

```diff
@@ -7,9 +7,9 @@
 
 #ifndef REDISTIMESERIES_MODULE_VERSION
 
-#define REDISTIMESERIES_VERSION_MAJOR 99
-#define REDISTIMESERIES_VERSION_MINOR 99
-#define REDISTIMESERIES_VERSION_PATCH 99
+#define REDISTIMESERIES_VERSION_MAJOR 1
```

Comment:
@filipecosta90 a bad merge from 1.4.5?

Reply:
your totally right. I saw it yesterday and had the idea that I force pushed the proper changes. Doing so :)

Reply:
@danni-m now fixed :+1: